src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java

```diff
@@ -37,36 +37,207 @@
 
 /**
  * <p>
- * A filter that uses nearby comments to suppress audit events.
+ * Filter {@code SuppressWithNearbyCommentFilter} uses individual comments to suppress audit events.
```

Comment:
I don't really agree with changing `nearby` to `individual`. `Nearby` is in the check name afterall.

Reply:
Reverted. Usually when xml and javadoc comments collide, I take the xml part.
Typically, xml comments are more up-to-date.

Reply:
> Typically, xml comments are more up-to-date.

I agree, but I also look at what we are losing from the Javadoc side. If there is something of value there, then we should save it and not just throw it away.